src/vs/workbench/browser/parts/editor/editorCommands.ts

```diff
@@ -1367,6 +1380,24 @@ function getEditorsContext(accessor: ServicesAccessor, resourceOrContext?: URI |
 	};
 }
 
+export function getEditorsFromContext(accessor: ServicesAccessor, resourceOrContext?: URI | IEditorCommandsContext, context?: IEditorCommandsContext): { editor: EditorInput; group: IEditorGroup }[] {
+	const { editors, groups } = getEditorsContext(accessor, resourceOrContext, context);
```

Comment:
Renaming to `resolveEditorsContext` and changing it such that the editors context is passed as argument and not retrieved. This should make it more clear and much nicer.

It will be used to following way now `resolveEditorsContext(getEditorsContext(accessor, resourceOrContext, context))`

Reply:
It turns `groupId` and `editorIndex` into `IEditorGroup` and `EditorInput`